web_scrapper/cron_file.txt

```diff
@@ -0,0 +1,2 @@
+# Execute after every 1 minute
+* * * * * python /home/nofal/PycharmProjects/TestPython/web_scrapper/main.py
```

Comment:
not sure if you're actually working in a virtualenv or not? But in case of a virtualenv the Python's executable path is given of virtualenv. e.g; if my virtualenv is p3 the path would be something like this:

 `/Users/naeemilyas/.virtualenvs/p3/bin/python`